### DISCUSSION TITLE
[Feature Request] Create a Private Fork of a Public Repo

### DISCUSSION THREAD

**Discussion by BenjaminYde** [community/community#59987] [Repositories]
### Select Topic Area

Product Feedback

### Body

Currently it is not possible to create a private fork of a public repo...
Now when you want to create a fork, you cannot set the visibility to private.
This should be possible...

**Comment by 3urobeat** [comment]
If GitHub blocks changing the visibility for forks with the reasoning "For security reasons, you cannot change the visibility of a fork." they probably won't change it.  
Although I have no idea what those "security reasons" might be.

**Comment by VladimirCreator** [comment]
Agree.

**Comment by smurp** [comment]
Even GPLed code should permit going private (to say nothing of other licenses) because it is the coder's responsibility to comply with any publication requirements, not GitHub's job to enforce them.
